isort/stdlibs/py313.py

```diff
@@ -0,0 +1,224 @@
+"""
+File contains the standard library of Python 3.13.
+
+DO NOT EDIT. If the standard library changes, a new list should be created
+using the mkstdlibs.py script.
+"""
+
+stdlib = {
+    "_ast",
+    "_thread",
+    "_tkinter",
+    "abc",
+    "aifc",
+    "argparse",
+    "array",
+    "ast",
+    "asynchat",
```

Comment:
Similarly, this list contains lots of modules that have been removed in 3.12 or earlier.

Was this list created using the mkstdlibs.py script?

Reply:
all changes to `stdlibbs/py*.py` were generated by `mkstdlibs.py` scripts